samples/BikeSharingApp/README.md

```diff
@@ -1,4 +1,100 @@
+
+    # Get the AKS resource id
+    az aks show -g $GROUP_NAME -n $AKS_NAME --query id -o tsv
+
+    # Create role assignment for access to AKS cluster
+    az role assignment create --assignee <clientId> --scope "<aksResourceId>" --role Contributor
+
+    # Get the container registry (ACR) id
+    az acr show -g $GROUP_NAME -n $ACR_NAME --query id -o tsv
+    
+    # Create role assignment for 'push access' to container registry
+    az role assignment create --assignee  <clientId>  --scope "<acrResourceId>" --role AcrPush
+    ```
+ -->
+
+
+## Deploy the BikeSharing sample app
+
+1. **Create a fork of this GitHub repo.** We need a *fork* because we'll be setting some GitHub Secrets to configure GitHub Actions in our workflow.
+
+1. **Clone your fork and navigate into its directory**.
+    ```bash
+    git clone https://github.com/USERNAME/dev-spaces
+    cd dev-spaces/samples/BikeSharingApp/
+    ```
+
+1. **Retrieve the HostSuffix for the `master` dev space.**
+    ```bash
+    azds show-context
+
+    Name                ResourceGroup     DevSpace  HostSuffix
+    ------------------  ----------------  --------  -----------------------
+    MyAKS               MyResourceGroup   master    fedcab0987.eus.azds.io
+    ```
+
+1. **Update the Helm chart with your HostSuffix.** Open [`charts/values.yaml`](https://github.com/Azure/dev-spaces/blob/master/samples/BikeSharingApp/charts/values.yaml) and replace all instances of `<REPLACE_ME_WITH_HOST_SUFFIX>` with the HostSuffix value you retrieved earlier. Save your changes and close the file.
+
+1. **Deploy the sample application to Kubernetes.** We'll use Helm to run this sample application, but other tooling could be used to run your entire application in a namespace within a cluster. The Helm commands are targeting the namespace named `master` you created earlier, and can take several minutes to complete.
+    ```bash
+    cd charts/
+    helm init --wait
+    helm install -n bikesharing . --dep-up --namespace master --atomic
+    ```
+    Note: **If you are using an RBAC-enabled cluster**, be sure to configure [a service account for Tiller](https://helm.sh/docs/using_helm/#role-based-access-control). Otherwise, `helm` commands will fail.
+
+1. **Open your browser to the app's website.** Navigate to the `bikesharingweb` service by opening the public URL from the `azds list-uris` command. In the below example, the public URL for the `bikesharingweb` service is http://master.bikesharingweb.fedcab0987.eus.azds.io/. Select **Aurelia Briggs (customer)** as the user, then select a bike to rent.
+    ```bash
+    azds list-uris
+
+    Uri                                                   Status
+    --------------------------------------------------    ---------
+    http://master.bikesharingweb.fedcab0987.eus.azds.io/  Available
+    http://master.gateway.fedcab0987.eus.azds.io/         Available
+    ```
+
+## Next Steps
+Now that you have the BikeSharing app deployed in AKS, try these walkthroughs:
```

Comment:
Add "Dev Spaces" to this line.  Either:
-Now that you have the BikeSharing app deployed in AKS with Dev Spaces, try these walkthroughs
OR
-Now that you have the BikeSharing app deployed in AKS, try these walkthroughs to learns how Dev Spaces can enhance the Kubernetes development experience: